arch/arm/boot/dts/overlays/README

```diff
@@ -2720,6 +2720,12 @@ Params: speed                   SPI bus speed (default 4000000)
         reset_pin               GPIO pin for RESET (default 25)
         height                  Display height (32 or 64; default 64)
 
+Name:   si446x-spi0
+Info:   Overlay for Si446x UHF Transceiver via SPI using si446x driver (currently out-of-tree at https://github.com/sunipkmukherjee/silabs.git)
```

Comment:
This line is far too long - there is a hard limit of 80 characters in this file.